specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-04-01-preview/kustoPool.json

```diff
@@ -0,0 +1,1534 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SynapseManagementClient",
+    "version": "2021-04-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.Synapse/kustooperations": {
+      "get": {
+        "tags": [
+          "KustoOperations"
+        ],
+        "operationId": "KustoOperations_List",
+        "x-ms-examples": {
+          "KustoOperationsList": {
+            "$ref": "./examples/KustoOperationsList.json"
+          }
+        },
+        "description": "Lists available operations for the Kusto sub-resources inside Microsoft.Synapse provider.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The operation was successful. The response contains the list of available operations.",
+            "schema": {
+              "$ref": "#/definitions/OperationListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
```

Comment:
>#/definitions/CloudError" [](start = 23, length = 25)

For all the error models, consider using the definition from common types here: ../../../../../common-types/resource-management/v2/types.json

Reply:
Replaced all